test/endToEnd/scenarioTest.js

```diff
@@ -112,7 +112,9 @@ function switchInDeterministic(browser) {
     .click('#deterministic-tab')
     .waitForElementVisible('#sensitivity-measurements-header')
     .click('#scenario-selector')
-    .click('option[label="Default"]')
+    .waitForElementVisible('#scenario-selector > option:nth-child(1)')
```

Comment:
nth-child

Reply:
label is scenario id. can put value on the component, but...